From 88b4cc717777aa88afbed8ec30b6ef95a7ec01d1 Mon Sep 17 00:00:00 2001
From: Christophe Giboudeaux <christophe@krop.fr>
Date: Mon, 21 Mar 2022 17:03:40 +0100
Subject: [PATCH] Don't try to install or use uic/rcc/designer copies.

These executables are installed by qtbase and qttools. Making copies
is not needed.
---
 sources/pyside2-tools/CMakeLists.txt | 32 ----------------------------
 1 file changed, 32 deletions(-)

diff --git a/sources/pyside2-tools/CMakeLists.txt b/sources/pyside2-tools/CMakeLists.txt
index ce65750..f9ebd84 100644
--- a/sources/pyside2-tools/CMakeLists.txt
+++ b/sources/pyside2-tools/CMakeLists.txt
@@ -55,38 +55,6 @@ else()
     set(DESIGNER_PATH "${TOOLS_PATH}/designer${EXE_EXT}")
 endif()
 
-install(FILES "${UIC_PATH}"
-        DESTINATION bin
-        PERMISSIONS
-        OWNER_EXECUTE OWNER_WRITE OWNER_READ
-        GROUP_EXECUTE GROUP_READ
-        WORLD_EXECUTE WORLD_READ)
-
-install(FILES "${RCC_PATH}"
-        DESTINATION bin
-        PERMISSIONS
-        OWNER_EXECUTE OWNER_WRITE OWNER_READ
-        GROUP_EXECUTE GROUP_READ
-        WORLD_EXECUTE WORLD_READ)
-
-if (EXISTS ${DESIGNER_PATH})
-    if (APPLE)
-        install(DIRECTORY "${DESIGNER_PATH}"
-                DESTINATION bin
-                FILE_PERMISSIONS
-                OWNER_EXECUTE OWNER_WRITE OWNER_READ
-                GROUP_EXECUTE GROUP_READ
-                WORLD_EXECUTE WORLD_READ)
-    else()
-        install(FILES "${DESIGNER_PATH}"
-                DESTINATION bin
-                PERMISSIONS
-                OWNER_EXECUTE OWNER_WRITE OWNER_READ
-                GROUP_EXECUTE GROUP_READ
-                WORLD_EXECUTE WORLD_READ)
-    endif()
-endif()
-
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/cmake_uninstall.cmake"
                "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake"
                IMMEDIATE @ONLY)
-- 
2.35.1

